bigtable/admin.go

```diff
@@ -1769,6 +1769,35 @@ func (ac *AdminClient) BackupInfo(ctx context.Context, cluster, backup string) (
 	return newBackupInfo(resp)
 }
 
+func (info *BackupInfo) Equal(info2 *BackupInfo) bool {
```

Comment:
This seems very Java-like to me. We don't typically have Equals methods in Go. Can we make use of `cmp` or add a testing helper instead?

Reply:
that worked thanks! Updated, PTAL @codyoss